package.json

```diff
@@ -11,6 +11,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/pickers": "^3.2.10",
     "@material-ui/styles": "^4.9.6",
+    "@mobiscroll/react-lite": "^4.10.8",
```

Comment:
You have installed this library but haven't used it anywhere

Reply:
Removed it